### PR TITLE
Event coupon dao test

### PR DESCRIPTION
**Pull request by mercado-carlos**

### CODE REVIEW

File: src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java [hunk 1]
```diff
@@ -17,60 +17,66 @@
...
+
+        @Test
+        public void couponRoutingTest() throws Exception {
+                this.mockMvc.perform(get("/admin-side/coupons"))
+                                .andExpect(status().isOk())
+                                .andExpect(forwardedUrl("/WEB-INF/jsp/coupons.jsp"));
+
+                this.mockMvc.perform(get("/admin-side/updateCoupon/1"))
+                                .andExpect(status().isOk())
+                                .andExpect(forwardedUrl("/WEB-INF/jsp/updateCouponDataForm.jsp"));
```
Comment (alberto-2-garcia) [inline src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java:45]: Is this test passing?

File: src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java [hunk 2]
```diff
@@ -17,60 +17,66 @@
...
+                                .andExpect(status().isOk())
+                                .andExpect(forwardedUrl("/WEB-INF/jsp/admin.jsp"));
+        }
+
+        // @Test
+        // public void searchByCartIDTest() throws Exception {
+
+        // this.mockMvc.perform(get("/admin-side/validateCoupon/1"))
+        // .andExpect(status().isOk());
+        // }
```
Comment (alberto-2-garcia) [inline src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java:35]: Why is this function commented?

File: src/test/java/com/backend/webproject/managers/AdminSideManagerTest.java [hunk 3]
```diff
@@ -0,0 +1,38 @@
+// package com.backend.webproject.managers;
+
+// import static org.junit.jupiter.api.Assertions.*;
```
Comment (alberto-2-garcia) [inline src/test/java/com/backend/webproject/managers/AdminSideManagerTest.java:3]: Why is this file commented?

File: src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java (hunk 1 above)
Comment (mercado-carlos) [reply src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java:45]: Yes it is already passing

File: src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java (hunk 2 above)
Comment (mercado-carlos) [reply src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java:35]: I commented this beacuse I am still working in that and it was giving problems when running Jacoco's report

File: src/test/java/com/backend/webproject/managers/AdminSideManagerTest.java (hunk 3 above)
Comment (mercado-carlos) [reply src/test/java/com/backend/webproject/managers/AdminSideManagerTest.java:3]: I commented this beacuse I am still working in that and it was giving problems when running Jacoco's report

File: src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java (hunk 1 above)
Comment (alberto-2-garcia) [reply src/test/java/com/backend/webproject/controllers/AdminSideControllerTest.java:45]: Okok

File: src/test/java/com/backend/webproject/managers/AdminSideManagerTest.java (hunk 3 above)
Comment (alberto-2-garcia) [reply src/test/java/com/backend/webproject/managers/AdminSideManagerTest.java:3]: Okok

**Pull request merged**